### DISCUSSION TITLE
Makefile and Dockerization of openMCT

### DISCUSSION THREAD

**Discussion by haisamido** [nasa/openmct#7556] [Show and tell]
I created this fork https://github.com/haisamido/openmct which allows one to simply run

`make openmct-start`

and then openmct builds a docker image and container and runs the npm service. Here's the draft [PR](https://github.com/nasa/openmct/pull/7557).

**Comment by milesbjackson** [comment]
Very intersted in this. Going to move mine to container so can run from docker file.

**Comment by akhenry** [comment]
@haisamido  Thanks for linking to your fork! We also have https://github.com/scottbell/openmct-quickstart.

_Official_ support for dockerized Open MCT is coming, I promise 😄
